no matching function for call to 'atomic_exchange' in manifold/src/properties.cpp:89:18

I'll try adding `#include <atomic>`

```
thirdparty/manifold/src/properties.cpp:89:18: error: no matching function for call to 'atomic_exchange'
      auto old = std::atomic_exchange(
                 ^~~~~~~~~~~~~~~~~~~~
/usr/local/lib/android/sdk/ndk/23.2.8568313/toolchains/llvm/prebuilt/linux-x86_64/bin/../sysroot/usr/include/c++/v1/atomic:1948:1: note: candidate template ignored: deduced conflicting types for parameter '_Tp' ('unsigned char' vs. 'int')
atomic_exchange(volatile atomic<_Tp>* __o, _Tp __d) _NOEXCEPT
^
/usr/local/lib/android/sdk/ndk/23.2.8568313/toolchains/llvm/prebuilt/linux-x86_64/bin/../sysroot/usr/include/c++/v1/atomic:1956:1: note: candidate template ignored: deduced conflicting types for parameter '_Tp' ('unsigned char' vs. 'int')
atomic_exchange(atomic<_Tp>* __o, _Tp __d) _NOEXCEPT
^
/usr/local/lib/android/sdk/ndk/23.2.8568313/toolchains/llvm/prebuilt/linux-x86_64/bin/../sysroot/usr/include/c++/v1/memory:5146:1: note: candidate template ignored: could not match 'shared_ptr' against 'atomic'
atomic_exchange(shared_ptr<_Tp>* __p, shared_ptr<_Tp> __r)
^
1 error generated.
scons: *** [thirdparty/manifold/src/properties.android.editor.arm64.o] Error 1
scons: building terminated because of errors.
[Time elapsed: 00:00:31.37]
```

https://github.com/godotengine/godot/actions/runs/11736862391/job/32696760939?pr=94321

--- a/src/properties.cpp
+++ b/src/properties.cpp
@@ -87,7 +87,8 @@ struct UpdateProperties {
       const int propVert = triProp[tri][i];
 
       auto old = std::atomic_exchange(
-          reinterpret_cast<std::atomic<uint8_t>*>(&counters[propVert]), 1);
+          reinterpret_cast<std::atomic<uint8_t>*>(&counters[propVert]),
+          static_cast<uint8_t>(1));
       if (old == 1) return;
 
       for (int p = 0; p < oldNumProp; ++p) {

--- a/src/smoothing.cpp
+++ b/src/smoothing.cpp
@@ -393,7 +393,7 @@ Vec<int> Manifold::Impl::VertHalfedge() const {
                auto old = std::atomic_exchange(
                    reinterpret_cast<std::atomic<uint8_t>*>(
                        &counters[halfedge_[idx].startVert]),
-                   1);
+                   static_cast<uint8_t>(1));
                if (old == 1) return;
                // arbitrary, last one wins.
                vertHalfedge[halfedge_[idx].startVert] = idx;

